hack/DebugWindowsNode.ps1

```diff
@@ -0,0 +1,79 @@
+Write-Output "Checking for common problems with Windows Kubernetes nodes"
+Î
```

Comment:
I think this is a stray character and needs to be removed.

Reply:
ahhhh windows copy paste error :)